mne/decoding/transformer.py

```diff
@@ -17,7 +18,10 @@
 
 
 class Scaler(TransformerMixin):
-    """Standardize data across channels.
+    u"""Standardize data across channels.
+
+    By default, this makes each time point (within each epoch) have
```

Comment:
> I suggest we move this discussion & correction to #3672; this PR is about updating the BaseEstimator.

Agreed. I'll keep the docstring and warning fixes in here, and let the "real" fixes be done in another PR.